python/langsmith/client.py

```diff
@@ -489,6 +501,16 @@ def __init__(
         # Create a session and register a finalizer to close it
         session_ = session if session else requests.Session()
         self.session = session_
+        self.compress_traces = os.getenv("LANGSMITH_COMPRESS_TRACES") == "true"
+        if self.compress_traces:
+            self.boundary = BOUNDARY
+            self.compressor: zstandard.ZstdCompressor = zstandard.ZstdCompressor()
```

Comment:
do we actually use `self.compressor` outside of instantiating `self.compressor_writer`?

Reply:
no we don't, removing